Factory.Cargo/TransportationFactory/AirTransportationFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Factory.Cargo.Factory;
+using Factory.Cargo.Transportation;
+
+namespace Factory.Cargo.TransportationFactory;
+
+public class AirTransportationFactory : ITransportationFactory
+{
+    private ITransportation? _transportation;
+
+    public ITransportation Create()
+        => _transportation ??= new AirTransportation();
```

Comment:
consider it is not thread safe, but for this case it is enough.

Reply:
Thanks @mohammadKarimi, I think in main implementation we should use atomic operations like Interlock.CompareExchange.
What is your idea?